projects/cdk/directives/auto-focus/handlers/ios.handler.ts

```diff
@@ -142,4 +150,31 @@ export class TuiIosAutofocusHandler extends AbstractTuiAutofocusHandler {
             element.style.setProperty(`height`, `100%`);
         }
     }
+
+    /**
+     * @note:
+     * inherit basic attributes values from real input
+     * for help iOS detect what do you want see on keyboard,
+     * for example [inputMode=numeric, autocomplete=cc-number]
+     */
+    private patchFakeInputFromFocusableElement(fakeInput: HTMLInputElement): void {
+        for (const property of [
```

Comment:
Let's move this array to external `const ATTRS = [` so it is not recreated each time and this function is shorter at a glance. Also I thought we prefer `forEach`.

Reply:
fixed